BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -47,7 +48,7 @@ open class BraintreeClient @VisibleForTesting internal constructor(
      */
     @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     val appLinkReturnUri: Uri?,
-) {
+) : APITiming {
```

Comment:
I think this is _technically_ a breaking change. We could make the case for it, but it also seems like maybe the API timing should live in the Analytics client? Could the `sendEvent` with timestamps method be moved there and be referenced from both `BraintreeClient` and `ConfigurationLoader`?